codalab/lib/bundle_cli.py

```diff
@@ -1455,43 +1455,50 @@ def copy_bundle(
             },
         )
 
-        # If bundle contents don't exist, finish after just copying metadata
+        # Collect information about how server should unpack
+        filename = nested_dict_get(source_info, 'metadata', 'name')
```

Comment:
Is there a reason to move this up here?

Reply:
Oh, actually, this shouldn't be here. I might have moved it around during testing/debuging.